### PR TITLE
fix(DB): Use sniffed values for Midsummer Music Doodad spawns

### LINKED ISSUE #12250: Midsummer - 3 Missing Midsummer Music Doodad (188174)
### Current Behaviour

There aren't any Midsummer Music Doodads in Stormwind City, Silvermoon City and The Exodar.

### Expected Blizzlike Behaviour

All Capital Cities' bonfires should have Music Doodads to play the Midsummer theme.

### Source

_No response_

### Steps to reproduce the problem

1. `.go xyz 9794.82 -7248.74 26.0974 530`
2. `.go xyz -8836.6 866.36 98.7168 0`
3. `.go xyz -3798.69 -11509 -134.821 530`

### Extra Notes

Values from TC:
Note: The Silvermoon gameobject was not working if placed in that location. I don't know if I should've modified the location slightly or just kept it that way, so I did not make a PR.
```
DELETE FROM `gameobject` WHERE `guid` IN (50554, 50553, 50552);
INSERT INTO `gameobject` (`guid`, `id`, `map`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`) VALUES 
(50554, 188174, 0, -8836.6, 866.36, 98.7168, 2.09439, 0.866025, 0.500001, 180, 100, 1),
(50553, 188174, 530, 9794.82, -7248.74, 26.0974, 3.68265, 0.96363, -0.267241, 180, 100, 1),
(50552, 188174, 530, -3798.69, -11509, -134.821, 5.06146, 0.573576, -0.819152, 180, 100, 1);
```

### AC rev. hash/commit

https://github.com/azerothcore/azerothcore-wotlk/commit/0ef7f999db7284c2a7776b70c9ecf008d8551f16

### Operating system

Windows 10

### Custom changes or Modules

_No response_

### PR DESCRIPTION
**Pull request by sudlud**
unfortunately one spawn is missing (guid 50564)

## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

This PR includes the proposed changes from the issue plus additional entries in game_event_gameobject.

To me it seems that the missing entries in game_event_gameobject led to Music not always beeing played as the GameObjectAI script checks if Midsummer event is active.
So without game_event_gameobject entries the GameObject immediately spawns - Midsummer event not active - GameObjectAI not playing music even if Midsummer Event gets started afterwards.

Therefore the proposed changes in the issue were correct but not complete.

## Issues Addressed:

- Closes #12250 

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [x] Sniffs (remember to share them with the open source community!)
- [ ] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [ ] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

.go xyz 9794.82 -7248.74 26.0974 530
.go xyz -3798.69 -11509 -134.821 530
.go xyz -8836.6 866.36 98.7168 0

before PR:
observe no Midsummer music playing

after PR:
observe Midsummer Music playing

## Known Issues and TODO List:

- [ ] unfortunately one spawn is missing (guid 50564)
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Kitzunu** [comment]
Are these spawns sniffed?

**Comment by sudlud** [comment]
> Are these spawns sniffed?

I took them from the proposed solution in the original issue.
They seem to taken from the TC DB.

**Comment by sudlud** [comment]
@Kitzunu I'll try to update this with sniffed spawns

**Comment by sudlud** [comment]
@Kitzunu I updated the PR with sniffed spawns, thanks to @heyitsbench

File: data/sql/updates/pending_db_world/rev_1697395883333045400.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+DELETE FROM `game_event_gameobject` WHERE `eventEntry` = 1 AND `guid` IN (50573, 50571, 50557, 50556, 50555, 50554, 50553, 50552);
+INSERT INTO `game_event_gameobject` (`eventEntry`,`guid`) VALUES
+(1, 50573),
+(1, 50571),
+(1, 50557),
+(1, 50556),
+(1, 50555),
+(1, 50554),
+(1, 50553),
+(1, 50552);
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1697395883333045400.sql:22]:
```suggestion
(1, 50552);

```

File: data/sql/updates/pending_db_world/rev_1697395883333045400.sql [hunk 2]
```diff
@@ -0,0 +1,22 @@
+-- Use sniffed values for Midsummer Music Doodad spawns
+DELETE FROM `gameobject` WHERE `guid` IN (50573, 50571, 50557, 50556, 50555, 50554, 50553, 50552);
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1697395883333045400.sql:2]: Use id here as well

**Review by Kitzunu — APPROVED** [review]

**Pull request merged**